.changeset/wise-terms-travel.md

```diff
@@ -0,0 +1,6 @@
+---
+'@aws-amplify/ui-react': patch
+'@aws-amplify/ui': patch
```

Comment:
Think we only need to patch `ui-react`:
```suggestion
```

Reply:
I  think we need both because we updated the base styles